ffi/cpp/examples/fixeddecimal-wasm/main.html

```diff
@@ -0,0 +1,1300 @@
+<!doctype html>
```

Comment:
Nit: Where did this giant HTML and JS file come from?  Please add a copyright header with more info, or replace it with a smaller file that you write yourself.

Reply:
I accidentally checked them in, they shouldn't be checked in. My bad.

Reply:
They all come from Emscripten, to be clear.

Reply:
Emscripten is kinda annoying to use, Rust's WASM story is _very good_ (and Rust moves its boilerplate into separate crates), emcc on the other hand produces a lot of boilerplate at once because it's nowhere near as good.